packages/code-du-travail-frontend/src/outils/IndemniteLicenciement/steps/Indemnite.js

```diff
@@ -55,10 +44,7 @@ function StepIndemnite({ form }) {
                   ))}
                 </Select>
                 {input.value && input.value.length > 0 && (
-                  <CancelButton
-                    variant="link"
-                    onClick={() => input.onChange("")}
-                  >
+                  <CancelButton onClick={() => input.onChange("")}>
```

Comment:
missing variant="link"

Reply:
It's not a link, it's a button !

Reply:
yep but it's secondary action that's why it's like that initially